src/server/pfs/server/driver.go

```diff
@@ -1461,35 +1454,83 @@ func (d *driver) createBranch(ctx context.Context, branch *pfs.Branch, commit *p
 				// possible that branch exists but has no head commit. This is fine, but
 				// branchInfo.Head must also be nil
 				if _, ok := err.(pfsserver.ErrNoHead); !ok {
-					return err
+					return fmt.Errorf("inspectCommit: %v", err)
```

Comment:
I'll change to what you suggest here, that seems much more descriptive.
I've adopted the general policy of leaving changes that I make when debugging to figure out where an error message came from since I figure those have some long term value.